.pre-commit-config.yaml

```diff
@@ -10,8 +10,7 @@ default_install_hook_types:
 
 # by default, confine hooks to these git stages (except for hooks that specify their own stages)
 # https://pre-commit.com/#top_level-default_stages
-default_stages:
-  - commit
+default_stages: [commit-msg]
```

Comment:
Same question as https://github.com/cal-itp/eligibility-server/pull/498/files#r1792376667

Reply:
Updated and force-pushed clean commit: https://github.com/cal-itp/calitp.org/pull/408/commits/04d3a37cdaa4f6c77c16bc2c72dc8792edbece8f